src/components/Feed/Socialize/Interactions.tsx

```diff
@@ -0,0 +1,136 @@
+import { VStack } from 'components/core/Spacer/Stack';
```

Comment:
this file captures this section 
<img width="234" alt="image" src="https://user-images.githubusercontent.com/6754223/195634037-23f7f8fe-cbd8-43a3-9fa1-5dfdf9e934c1.png">

there's a good chunk of logic just to determine what combination of comments / admires to show